app/assets/stylesheets/application.css.scss

```diff
@@ -3,10 +3,17 @@
 */
 @import "bootstrap-sprockets";
 @import "bootstrap";
+@import url(https://fonts.googleapis.com/css?family=Source+Sans+Pro:400,200,200italic,300,300italic,400italic,600);
```

Comment:
Similar deal -- would prefer this get loaded in the asset pipeline rather than the live internet. HMU on slack if you need a hand with this.